test/e2e/network/netpol/kubemanager.go

```diff
@@ -115,11 +124,16 @@ func (k *kubeManager) probeConnectivity(nsFrom string, podFrom string, container
 	var cmd []string
 	switch protocol {
 	case v1.ProtocolSCTP:
-		cmd = []string{"/agnhost", "connect", net.JoinHostPort(addrTo, port), "--timeout=1s", "--protocol=sctp"}
+		cmd = []string{"/agnhost", "connect", net.JoinHostPort(addrTo, port), "--timeout=3s", "--protocol=sctp"}
 	case v1.ProtocolTCP:
-		cmd = []string{"/agnhost", "connect", net.JoinHostPort(addrTo, port), "--timeout=1s", "--protocol=tcp"}
+		cmd = []string{"/agnhost", "connect", net.JoinHostPort(addrTo, port), "--timeout=3s", "--protocol=tcp"}
 	case v1.ProtocolUDP:
-		cmd = []string{"/agnhost", "connect", net.JoinHostPort(addrTo, port), "--timeout=1s", "--protocol=udp"}
+		cmd = []string{"/agnhost", "connect", net.JoinHostPort(addrTo, port), "--timeout=3s", "--protocol=udp"}
+		//// TODO: @perithompson test whether udp polling can work on windows with agnhost
+		if framework.NodeOSDistroIs("windows") {
+			framework.Failf("not supported udp tests for windows")
+			return false, "no udp for windows", fmt.Errorf("udp on windows unsupported")
```

Comment:
do you mean skip?, otherwise is better to let it fail so you have signal when the problem has been fixed , instead of having to remove this code ... this is not really nice honestly, failing a test based on a distro

Reply:
as far as im concerned theyre equivalent.  Anyone testing windows will be able to leverage skip or fail in largely the same way. 

if you feel strongly ill change it to skip, but the priority would be to fix UDP so i havent thought deeply about this :)

Reply:
i modified this to a log.warn.  It is ok for now, that will make the review easier i think